changed the header to be Pascal on CommuteDistanceDistribution component

FaneTree opened the pull request:
## Overview of the Pull Request:
_Describe your changes here..._

## link to Trello card or Github issue

## How Has This Been Tested?
- Please describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.  => Change only lines 26 and 40 about "Distribution of Commute Distance" on CommuteDistanceDistribution component
- If necessary, please also list any relevant details for your test setup/configuration

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [ ] Have you written instructions on how to test that your changes work as expected?
- [ ] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

FaneTree committed:
changed the header to be Pascal on CommuteDistanceDistribution component

OlaShabalina committed:
Merge branch 'staging' into staging

OlaShabalina commented:
Hi @FaneTree , congrats on first commit! 

What about the following charts?
![image](https://user-images.githubusercontent.com/88268603/232463380-52c66fa1-415a-48eb-97ea-f07405e5c9f6.png)

Also, would you be able to revert the changes to `package.json` and `package-lock.json`, those would need to be done in a separate PRs to keep the code single purposed.

OlaShabalina commented on package.json:
```diff
@@ -20,7 +20,7 @@
     "highcharts": "^10.3.3",
     "highcharts-react-official": "^3.2.0",
-    "next": "^13.2.3",
+    "next": "^13.3.0",
```
```suggestion
    "next": "^13.2.3",
```

FaneTree committed:
Update package.json

Thank you for your request. 
Next time I will be more aware of version differences.

Co-authored-by: Olga <88268603+OlaShabalina@users.noreply.github.com>

Amutha37 reviewed (approved):
Great first commit Fane.

OlaShabalina reviewed (changes requested):
Could you please revert changes to `package-lock.json` as well? Just the code that is needed for this task.

FaneTree committed:
minor change in CommuteDistanceDistribution TripCountAndTravelMethods to make Pascal case

FaneTree committed:
Merge branch 'staging' of github.com:FaneTree/council-emissions-calculator into staging

FaneTree commented:
I have changed all as discussed. To summarise all the changes.
- Adjusted in CommuteDistanceDistribution component
    - line 26 (For header before description),
    - line 40 (for chart header)
- Adjusted in TripCountAndTravelMethods component
    - line 44 Trip Count and Travel Methods ⇒ Count
    - line 69 "Active-public-shared Travel” ⇒ Travel
    - line 81 "Individual Travel” ⇒ Travel

Note: 
1. checked <Text> CSS and both are the same for “Travel to Work Days” and “Distribution of Commute Distance”.
2. pull the current version again to align the version and now the local and this repository are the same.

OlaShabalina commented:
Missed 1:
![image](https://user-images.githubusercontent.com/88268603/232716146-77cf5eef-06a1-45e6-a636-4cee07212b46.png)

hqtan commented:
Awesome work on your first PR to this repo @FaneTree ! 🙌🏽 

You're _almost_ done with this PR; remember to address @OlaShabalina 's comment about remove `package*.json` files k?

🙇🏽

FaneTree committed:
minor adjust to add back the last space in package and package-lock

FaneTree committed:
minor adjust to add back the last space in package and package-lock

OlaShabalina reviewed (approved):
Well done @FaneTree , looks good to me! 👍🏻
